### PR TITLE
Support predict and evaluate on Imageset

### PR DESCRIPTION
**Pull request by hkvision**
Fix https://github.com/intel-analytics/zoo/issues/270

Need to depend on BigDL 0.6.0 and use its `evaluateImage` API

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/models/Topology.scala [hunk 1]
```diff
@@ -326,6 +333,10 @@ abstract class KerasNet[T: ClassTag](implicit ev: TensorNumeric[T])
     localPredictor.predict(x)
   }
 
+  def predict(x: ImageSet): ImageSet = {
+    ImageSet.fromImageFrame(predictImage(x.toImageFrame()))
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/models/Topology.scala]: In predictImage, the argument is `batchPerPartition`, any suggestion on how to use batchSize here... seems we can't get the partition number.

**Comment by dding3** [comment]
LGTM

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/models/Topology.scala (hunk 1 above)
Comment (dding3) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/models/Topology.scala]: I think partition number can be get by `rdd.partitions.length`

**Comment by hkvision** [comment]
jenkins passed http://10.239.47.211:18888/job/ZOO-PR-Validation/848/

**Comment by hkvision** [comment]
Thanks so much for your review and help!

**Pull request merged**